Ninano/Ninano/Views/CalendarTab/WeeklyCalendar/CategoryCell.swift

```diff
@@ -8,7 +8,7 @@
 import UIKit
 
 class CategoryCell: UITableViewCell {
-    var viewModels: [SearchEventCellViewModel] = [] {
+    var searchEvent: [SearchEvent] = [] {
```

Comment:
```suggestion
    var eventList: [Event] = [] {
```
위에서 언급한 것과 동일한 내용이에요 ..! Rain의 생각도 궁금합니다

Reply:
Event로 수정했습니다~